channel-adapter/channel-adapter.bal

```diff
@@ -0,0 +1,19 @@
+import ballerina/io;
+import ballerinax/jira;
+
+configurable string username = "ballerina@com.baclmock.io";
+configurable string password = "SecureP@ssw0rd!";
```

Comment:
this looks unsafe. let's put "admin" "admin" as username and password and add a comment saying `// provide actual credential at execution`